dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -143,6 +143,16 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
         return AppModel::Identity::IsPackagedProcess() && IsBackgroundTaskBuilderAvailable();
     }
 
+    inline bool IsElevated()
+    {
+        wil::unique_handle processToken;
+        THROW_IF_WIN32_BOOL_FALSE(OpenProcessToken(GetCurrentProcess(), MAXIMUM_ALLOWED, &processToken));
+
+        auto tokenLabel{ wil::get_token_information<TOKEN_MANDATORY_LABEL>(processToken.get()) };
+        DWORD subAuthority = static_cast<DWORD>(*GetSidSubAuthorityCount(tokenLabel->Label.Sid) - 1);
+        return *GetSidSubAuthority(tokenLabel->Label.Sid, subAuthority) >= SECURITY_MANDATORY_HIGH_RID;
```

Comment:
For elevated, can you add a comment that this equates to HIGH_RID? SYSTEM is not applicable right for elevated?

Reply:
This just blocks all elevated + system callers. I can just use == instead. If that is preferred.